src/gui/resources/demoprofile.conf

```diff
@@ -629,4 +629,4 @@ CurrentMode={11C11AE3-3195-4DFC-B8AC-2FEA703414E5}
 HwModified=40b6f054
 ModeCount=3
 Modified=2015a5b9
-Name=ckb Demo
+Name=ckb-next Demo
```

Comment:
I think it might be better if we simplify this to just "Demo".

Reply:
Ok.